src/pages/specialHotPage.ts

```diff
@@ -0,0 +1,21 @@
+import { Page } from "@playwright/test";
+
+export default class SpecialHotPage{
+
+
+    constructor(public page: Page) { }
+
+    async addFirstProductToTheCart() { 
+        await this.page.hover("//div[@class='image']/a", {

```

Comment:
locators should be in a separate variables

Reply:
Done